### PR TITLE
Secured service IdentifyTask iOS sample

### PR DESCRIPTION
**Pull request by gdhakal**
Secured service IdentifyTask iOS sample

### CODE REVIEW

File: runtime-ios/secured-service-identify/IdentifyTaskSampleViewController.m [hunk 1]
```diff
@@ -0,0 +1,177 @@
+// Copyright 2012 ESRI
```
Comment (jgravois) [inline runtime-ios/secured-service-identify/IdentifyTaskSampleViewController.m]: this should probably be '2015'

**Comment by jgravois** [comment]
thanks for this contribution @gdhakal!  can we get a volunteer thats not skurred of iOS to run the sample code and provide feedback?

**Comment by NoashX** [comment]
on it, will test tomorrow

**Comment by NoashX** [comment]
the code works like a charm, and the readme is clear with relevant links, good work @gdhakal !

Merging #29

**Pull request merged**